gov_uk_dashboards/components/plotly/heading.py

```diff
@@ -0,0 +1,30 @@
+"""heading components"""
+from enum import Enum
+
+from dash import html
+
+
+class HeadingSizes(str, Enum):
+    """Enum class to specify the allowed paragraph sizes"""
+
+    EXTRA_LARGE = "govuk-heading-xl"
+    LARGE = "govuk-heading-l"
+    MEDIUM = "govuk-heading-m"
+    SMALL = "govuk-heading-s"
+
+
+def heading1(text: str, size: HeadingSizes = HeadingSizes.LARGE) -> html.H1:
+    """Return a H1 dash component"""
+    return html.H1(text, className=size)
+
+
+def heading2(text: str, size: HeadingSizes = HeadingSizes.MEDIUM) -> html.H2:
+    """Return a H2 dash component"""
+
+    return html.H2(text, className=size)
+
+
+def heading3(text: str, size: HeadingSizes = HeadingSizes.SMALL) -> html.H3:
```

Comment:
> We were using this page for defaults, and didn't think we'd need a H4 https://design-system.service.gov.uk/styles/typography/#headings yet

If thats the case you need to remove the need to override the heading size then right?  Because you not meant to use XL for h2

Reply:
@chopkinsmade Should HeadingSizes be restricted to   MEDIUM = "govuk-heading-m" & SMALL = "govuk-heading-s" only

Reply:
Additionally, their usage examples do see all of the components taking on different classes, depending whether you want to use an XL size or not - so think we need this functionality to replicate.